modules/osd/index.ts

```diff
@@ -38,20 +38,6 @@ const handleReveal = (self: any, property: string) => {
     })
 }
 
-const getPosition = (pos: OSDAnchor): ("top" | "bottom" | "left" | "right")[] => {
-    const positionMap: { [key: string]: ("top" | "bottom" | "left" | "right")[] } = {
-        "top": ["top"],
-        "top right": ["top", "right"],
-        "top left": ["top", "left"],
-        "bottom": ["bottom"],
-        "bottom right": ["bottom", "right"],
-        "bottom left": ["bottom", "left"],
-        "right": ["right"],
-        "left": ["left"],
```

Comment:
In the process of consolidating the getPosition function, it seems that you forgot to include the `right` and `left` properties.